aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -244,8 +242,7 @@ module aptos_framework::account {
         };
 
         // Derive the authentication key of the new PK
-        vector::push_back(&mut new_pk_bytes, 0);
-        let new_auth_key = hash::sha3_256(new_pk_bytes);
+        let new_auth_key = ed25519::public_key_bytes_to_authentication_key(new_pk_bytes);
```

Comment:
Heads up @chloeqjz and @davidiw!

Reply:
Uhm... don't follow. How are we computing it twice? I'm replacing the old code that computed it with this new function call that computes it.

Reply:
Oh... because of the same call `public_key_bytes_to_authentication_key` being done later below...

Reply:
The second let `public_key_bytes_to_authentication_key` call was on a different argument...

Reply:
Anyway, addressed. Perhaps take another look and let me know?